cl/corpus_importer/tasks.py

```diff
@@ -2005,12 +2005,16 @@ def download_pacer_pdf_by_rd(
     s = ProxyPacerSession(
         cookies=session_data.cookies, proxy=session_data.proxy_address
     )
-    report = FreeOpinionReport(pacer_court_id, s)
-
-    r, r_msg = report.download_pdf(
-        pacer_case_id, pacer_doc_id, magic_number, de_seq_num=de_seq_num
-    )
-
+    if is_appellate_court(rd.docket_entry.docket.court_id):
```

Comment:
Can't we just do `pacer_court_id` here? Or do the `map_to_cl_id` tweak, instead of the DB hit?

Reply:
> Can't we just do `pacer_court_id` here?

Yes, we should do this. I'll update the if statement